application/backend/src/main/resources/libs/bulma/components/contactform/contactform.html

```diff
@@ -17,39 +17,51 @@
 */-->
 
 <sly data-sly-use.model="pl.ds.bulma.components.models.ContactFormComponent">
-  <form>
+  <form data-config-captcha-public-key='${model.captchaPublicKey}'
+        data-config-http-end-point='${model.configEndpoint}'>
```

Comment:
We have this configuration added to the body: https://github.com/websight-io/bulma/blob/main/application/backend/src/main/resources/libs/bulma/components/page/body.html#L23

We can remove it from the body, but we need to update js files as well to use both configs from the form definition